ci/courtesy-push.yml

```diff
@@ -44,6 +49,12 @@ steps:
     cd ..\azure-pipelines-tasks\ci\courtesy-push
     npm install
     node open-courtesy-push-pull-request.js $releaseBranch
-  displayName: Create PR in Azure DevOps
+
+    # Send message to slack channel
+    $body = '{"text": "Created tasks courtesy push PR. Someone please approve/merge this :please-puss-in-boots: ' + $env:PrLink + '"}'
+    Invoke-RestMethod -Uri $slackUri -Method Post -Body $body -ContentType 'application/json'
```

Comment:
Where is $slackUri defined?

Reply:
Its a secret in the pipeline definition

Reply:
It does work - so far I've just been sending things to #damccorm-playground though